client-src/src/actions/fetch-service-type-instances.js

```diff
@@ -0,0 +1,20 @@
+import axios from 'axios';
+import settings from '../settings';
+
+export default function fetchServicetypeInstances(e) {
```

Comment:
let's remove this so that we send the actual value to the function, not the whole event... it was my mistake I know 😞